static/js/apps/explore/app.tsx

```diff
@@ -229,7 +229,7 @@ export function App(props: { isDemo: boolean }): JSX.Element {
       }
     }
     const userMessage = {
-      msg: fulfillData["userMessage"] || "",
+      msg: !_.isEmpty(fulfillData["userMessages"]) ? fulfillData["userMessages"][0] : "",
```

Comment:
this is a temporary change to make sure UI doesn't break if this PR gets merged before the UI changes (it will be updated by the UI changes in a separate PR).